tools/syz-agent/agent.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2025 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/google/syzkaller/pkg/agent"
+	_ "github.com/google/syzkaller/pkg/agent/flow/patching"
+	"github.com/google/syzkaller/pkg/tool"
+	"google.golang.org/adk/session"
+)
+
+func main() {
+	var (
+		flagFlow       = flag.String("workflow", "", "workflow to execute")
+		flagInput      = flag.String("input", "", "input json file with workflow arguments")
```

Comment:
I think one thing we need (which syzbot doesn't provide programmatically) is the kernel image and disk image to repro bugs.

What's on the JSON file anyway?

Reply:
> What's on the JSON file anyway?

It's the inputs object for the specified pipeline. For the patching pipeline it's serialized `patching.Inputs` struct.